README.md

```diff
@@ -28,7 +28,7 @@ rsnapshot_master_ssh_key: id_rsa
 
 # User used by the rsnapshot master to connect on the configured host
 rsnapshot_slave_user: backupuser
-rsnapshot_slave_user_password: False #If set, use an unencrypted value and store it in vault
+rsnapshot_slave_userpassword: False #If set, use an unencrypted value and store it in vault
```

Comment:
Why removing the `_`? `rsnapshot_slave_user_password` was perfectly fine :) Your error was that there was no `rsnapshot_slave_user_password` variable defined in `defaults/main.yml`.

Reply:
Oups sorry, didn't see your comment...